src/style.css

```
@@ -1,15 +1,6 @@
 @font-face {
   font-family: Avenir;
   src: url(fonts/Avenir-Book.ttf);
-  font-style: normal;
-  font-weight: 400;
-}
-
-@font-face {
-  font-family: Avenir-Heavy;
-  src: url(fonts/Avenir-Heavy.ttf);
-  font-style: normal;
-  font-weight: 900;
 }
```

```suggestion
@font-face {
  font-family: Avenir;
  src: url(fonts/Avenir-Book.ttf);
  font-style: normal;
  font-weight: 300;
}

@font-face {
  font-family: Avenir;
  src: url(fonts/Avenir-Heavy.ttf);
  font-style: normal;
  font-weight: 600;
}
```
and change `font-weight` in places you need